linuxdeploy-plugin-gtk.sh

```diff
@@ -305,3 +308,19 @@ for directory in "${PATCH_ARRAY[@]}"; do
         ln $verbose -s "${file/\/usr\/lib\//}" "$APPDIR/usr/lib"
     done < <(find "$directory" -name '*.so' -print0)
 done
+
+# set write permission on lib64 again to make it deletable.
+chmod +w "$APPDIR"/usr/lib64 || true
```

Comment:
`/usr/lib64` is 755 on systems. I think the issue is how you create this directory before calling linuxdeploy.